.github/workflows/tests.yml

```diff
@@ -61,6 +61,24 @@ jobs:
       # --- Set-up ---
       - name: "Ping redis"
         run: redis-cli -h redis ping
+      # --- Cache based on Conan version and ExternalProjects cmake source
+      - name: Get Conan version
+        id: get-conan-version
+        run: |
+          echo "::set-output name=conan-version::$(conan --version | tr -d '[:alpha:] ')"
+        shell: bash
```

Comment:
Rather than repeating `shell: bash` on these commands, can we set `shell: bash` in defaults? May apply to other workflow files.

Reply:
yes, i did not know it but you should be able to do [this](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-5):
```yaml
defaults:
  run:
    shell: bash
```